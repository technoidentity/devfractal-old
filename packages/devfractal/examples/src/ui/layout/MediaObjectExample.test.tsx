import React from 'react'
import renderer, { ReactTestRenderer } from 'react-test-renderer'
import { MediaObject } from './MediaObjectExample'

test('MediaObject Example', () => {
  const component: ReactTestRenderer = renderer.create(<MediaObject />)
  expect(component.toJSON()).toMatchSnapshot()
})
