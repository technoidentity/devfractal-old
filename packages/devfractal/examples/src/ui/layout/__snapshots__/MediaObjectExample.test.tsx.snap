// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MediaObject Example 1`] = `
Array [
  <section
    className="section"
  >
    <h1
      className="title"
    >
      Simple Media Object
    </h1>
    <article
      className="media"
    >
      <figure
        className="media-left"
      >
        <figure
          className="image is-64x64"
        >
          <img
            className=""
            src="https://bulma.io/images/placeholders/128x128.png"
          />
        </figure>
      </figure>
      <div
        className="media-content"
      >
        <div
          className="field"
        >
          <div
            className="control"
          >
            <textarea
              className="textarea"
              placeholder="Add a comment..."
            />
          </div>
        </div>
        <nav
          className="level"
        >
          <div
            className="level-item level-left"
          >
            <div
              className="control"
            >
              <button
                className="button is-info"
                type="button"
              >
                Submit
              </button>
            </div>
          </div>
          <div
            className="level-item level-right"
          >
            <div
              className="control"
            >
              <label
                className="checkbox"
              >
                <input
                  className="checkbox"
                  type="checkbox"
                />
                Press enter to submit
              </label>
            </div>
          </div>
        </nav>
      </div>
    </article>
  </section>,
  <section
    className="section"
  >
    <h1
      className="title"
    >
      Media Object
    </h1>
    <article
      className="media"
    >
      <figure
        className="media-left"
      >
        <figure
          className="image is-64x64"
        >
          <img
            className=""
            src="https://bulma.io/images/placeholders/128x128.png"
          />
        </figure>
      </figure>
      <div
        className="media-content"
      >
        <div
          className="content"
        >
          <p>
            <strong>
              John Smith
            </strong>
             
            <small>
              @johnsmith
            </small>
             
            <small>
              31m
            </small>
            <br />
            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Proin ornare magna eros, eu pellentesque tortor vestibulum ut. Maecenas non massa sem. Etiam finibus odio quis feugiat facilisis.
          </p>
        </div>
        <nav
          className="level is-mobile"
        >
          <div
            className="level-item level-left"
          >
            <span
              className="icon"
            >
              <svg
                aria-hidden="true"
                className="svg-inline--fa fa-reply fa-w-16 "
                data-icon="reply"
                data-prefix="fas"
                focusable="false"
                role="img"
                style={Object {}}
                viewBox="0 0 512 512"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M8.309 189.836L184.313 37.851C199.719 24.546 224 35.347 224 56.015v80.053c160.629 1.839 288 34.032 288 186.258 0 61.441-39.581 122.309-83.333 154.132-13.653 9.931-33.111-2.533-28.077-18.631 45.344-145.012-21.507-183.51-176.59-185.742V360c0 20.7-24.3 31.453-39.687 18.164l-176.004-152c-11.071-9.562-11.086-26.753 0-36.328z"
                  fill="currentColor"
                  style={Object {}}
                />
              </svg>
            </span>
            <span
              className="icon"
            >
              <svg
                aria-hidden="true"
                className="svg-inline--fa fa-retweet fa-w-20 "
                data-icon="retweet"
                data-prefix="fas"
                focusable="false"
                role="img"
                style={Object {}}
                viewBox="0 0 640 512"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M629.657 343.598L528.971 444.284c-9.373 9.372-24.568 9.372-33.941 0L394.343 343.598c-9.373-9.373-9.373-24.569 0-33.941l10.823-10.823c9.562-9.562 25.133-9.34 34.419.492L480 342.118V160H292.451a24.005 24.005 0 0 1-16.971-7.029l-16-16C244.361 121.851 255.069 96 276.451 96H520c13.255 0 24 10.745 24 24v222.118l40.416-42.792c9.285-9.831 24.856-10.054 34.419-.492l10.823 10.823c9.372 9.372 9.372 24.569-.001 33.941zm-265.138 15.431A23.999 23.999 0 0 0 347.548 352H160V169.881l40.416 42.792c9.286 9.831 24.856 10.054 34.419.491l10.822-10.822c9.373-9.373 9.373-24.569 0-33.941L144.971 67.716c-9.373-9.373-24.569-9.373-33.941 0L10.343 168.402c-9.373 9.373-9.373 24.569 0 33.941l10.822 10.822c9.562 9.562 25.133 9.34 34.419-.491L96 169.881V392c0 13.255 10.745 24 24 24h243.549c21.382 0 32.09-25.851 16.971-40.971l-16.001-16z"
                  fill="currentColor"
                  style={Object {}}
                />
              </svg>
            </span>
            <span
              className="icon"
            >
              <svg
                aria-hidden="true"
                className="svg-inline--fa fa-heart fa-w-16 "
                data-icon="heart"
                data-prefix="fas"
                focusable="false"
                role="img"
                style={Object {}}
                viewBox="0 0 512 512"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M462.3 62.6C407.5 15.9 326 24.3 275.7 76.2L256 96.5l-19.7-20.3C186.1 24.3 104.5 15.9 49.7 62.6c-62.8 53.6-66.1 149.8-9.9 207.9l193.5 199.8c12.5 12.9 32.8 12.9 45.3 0l193.5-199.8c56.3-58.1 53-154.3-9.8-207.9z"
                  fill="currentColor"
                  style={Object {}}
                />
              </svg>
            </span>
          </div>
        </nav>
      </div>
      <div
        className="media-right"
      >
        <a
          className="delete"
        />
      </div>
    </article>
  </section>,
  <section
    className="section"
  >
    <h1
      className="title"
    >
      Nesting
    </h1>
    <article
      className="media"
    >
      <figure
        className="media-left"
      >
        <figure
          className="image is-64x64"
        >
          <img
            className=""
            src="https://bulma.io/images/placeholders/128x128.png"
          />
        </figure>
      </figure>
      <div
        className="media-content"
      >
        <div
          className="content"
        >
          <strong>
            Barbara Middleton
          </strong>
          <br />
          Lorem ipsum dolor sit amet, consectetur adipiscing elit. Duis porta eros lacus, nec ultricies elit blandit non. Suspendisse pellentesque mauris sit amet dolor blandit rutrum. Nunc in tempus turpis.
          <br />
          <small>
            <a
              href="#!"
            >
              Like
            </a>
             路 
            <a
              href="#!"
            >
              Reply
            </a>
             路 3 hrs
          </small>
        </div>
        <article
          className="media"
        >
          <figure
            className="media-left"
          >
            <figure
              className="image is-32x32"
            >
              <img
                className=""
                src="https://bulma.io/images/placeholders/128x128.png"
              />
            </figure>
          </figure>
          <div
            className="media-content"
          >
            <div
              className="content is-medium"
            >
              <strong>
                Sean Brown
              </strong>
              <br />
              Lorem ipsum dolor sit amet, consectetur adipiscing elit. Duis porta eros lacus, nec ultricies elit blandit non. Suspendisse pellentesque mauris sit amet dolor blandit rutrum. Nunc in tempus turpis.
              <br />
              <small>
                <a
                  href="#!"
                >
                  Like
                </a>
                 路 
                <a
                  href="#!"
                >
                  Reply
                </a>
                 路 2 hrs
              </small>
            </div>
            <article
              className="media"
            >
              Vivamus quis semper metus, non tincidunt dolor. Vivamus in mi eu lorem cursus ullamcorper sit amet nec massa.
            </article>
            <article
              className="media"
            >
              Morbi vitae diam et purus tincidunt porttitor vel vitae augue. Praesent malesuada metus sed pharetra euismod. Cras tellus odio, tincidunt iaculis diam non, porta aliquet tortor.
            </article>
            <article
              className="media"
            >
              <figure
                className="media-left"
              >
                <figure
                  className="image is-64x64"
                >
                  <img
                    className=""
                    src="https://bulma.io/images/placeholders/128x128.png"
                  />
                </figure>
              </figure>
              <div
                className="media-content"
              >
                <div
                  className="content"
                >
                  <strong>
                    Kayli Eunice
                  </strong>
                  <br />
                  Sed convallis scelerisque mauris, non pulvinar nunc mattis vel. Maecenas varius felis sit amet magna vestibulum euismod malesuada cursus libero. Vestibulum ante ipsum primis in faucibus orci luctus et ultrices posuere cubilia Curae; Phasellus lacinia non nisl id feugiat.
                  <br />
                  <small>
                    <a
                      href="#!"
                    >
                      Like
                    </a>
                     路 
                    <a
                      href="#!"
                    >
                      Reply
                    </a>
                     路 2 hrs
                  </small>
                </div>
              </div>
            </article>
            <article
              className="media"
            >
              <figure
                className="media-left"
              >
                <figure
                  className="image is-48x48"
                >
                  <img
                    className=""
                    src="https://bulma.io/images/placeholders/128x128.png"
                  />
                </figure>
              </figure>
              <div
                className="media-content"
              >
                <div
                  className="field"
                >
                  <div
                    className="control"
                  >
                    <textarea
                      className="textarea"
                    />
                  </div>
                </div>
                <div
                  className="field"
                >
                  <div
                    className="control"
                  >
                    <button
                      className="button"
                      type="button"
                    >
                      Post comment
                    </button>
                  </div>
                </div>
              </div>
            </article>
          </div>
        </article>
      </div>
    </article>
  </section>,
]
`;
